src/Microsoft.Health.Dicom.Functions/Update/UpdateDurableFunction.Orchestration.cs

```diff
@@ -327,8 +185,17 @@ public partial class UpdateDurableFunction
         {
             // We don't want to populate all the errors in Azure Table Storage, DTFx may attempt to compress the entry as needed using GZip and storing in blob storage
             // But I think we should also be wary of what the user experience is for this via the response, so restricting to 5 errors for now. We can update based on feedback.
-            // TODO: Inform the user that the remaining failures can be found in the logs.
             errors.AddRange(instanceErrors.Take(5));
+
+            if (instanceErrors.Count > 5)
+            {
+                errors.Add("There are more instances failed to update than listed above. Please check the diagnostics logs for the complete list.");
```

Comment:
Is it worth even having this "top 5" logic in the response? Should we just put all of the daa in the logs?

Reply:
We are not sure, how many enabled diagnostic logging. I would think this will be a step for them to do if they didnt do